src/lib/drivers/accelerometer/CMakeLists.txt

```diff
@@ -32,4 +32,8 @@
 ############################################################################
 
 px4_add_library(drivers_accelerometer PX4Accelerometer.cpp)
-target_link_libraries(drivers_accelerometer PRIVATE drivers__device)
+target_link_libraries(drivers_accelerometer
+	PRIVATE
+		drivers__device
+		mathlib
```

Comment:
I guess adding the dependency here is not directly related, right?

Reply:
The dependency was always there, but happened to be in the right linking order previously.